src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebVersionRouteConfig.kt

```diff
@@ -24,21 +21,26 @@ object WebVersionRouteConfig : RouteConfig
                     VersionController::class, "getZippedByNameAndVersion", contentType = ContentTypes.binarydata)
                     .secure(readReports),
             WebEndpoint("/report/:name/version/:version/publish/",
-                    org.vaccineimpact.orderlyweb.controllers.api.ReportController::class, "publish",
+                    ReportController::class, "publish",
                     method = HttpMethod.post)
                     .transform()
                     .json()
                     .secure(reviewReports),
             WebEndpoint("/report/:name/version/:version/data/:data/",
-                    org.vaccineimpact.orderlyweb.controllers.api.DataController::class, "downloadData",
+                    DataController::class, "downloadData",
                     contentType = ContentTypes.binarydata)
                     .secure(readReports),
+            WebEndpoint("/report/:name/version/:version/tag/",
+                    org.vaccineimpact.orderlyweb.controllers.web.ReportController::class, "tagVersion")
+                    .json()
+                    .post()
+                    .secure(reviewReports),
             WebEndpoint("/report/:name/version/:version/tags/",
```

Comment:
Just realised that elsewhere with web routes we've gone with singular rather than plurals - should this be /tag/ ?

Reply:
Where have we got singulars when we're actually getting multiple things? We've got /roles and /permissions for example..

Reply:
Although this is a POST rather than a GET. I think 'tag' would be a bit confusing though..

Reply:
Oh yeah, hadn't noticed roles and permissions. I guess all the page urls are singular but everything else is plural? In which case fine, but then I should probably change the new url to /tags/ plural as well.

Reply:
Oh I see, yes they should be consistent :+1: